app/view/Table.svelte

```diff
@@ -46,10 +46,28 @@ $: expanded = false; // why would this expand?
   left: 100%;
   top: 0;
   bottom: 0;
-  width: calc(100vw - 350px);
+  z-index: 1;
+  width: calc(100% - 350px);
   transform: translateX(-350px);
   transition: transform 0.2s;
   background-color: blue;
+  background-size: 20%;
+
+  &.marquise {
+    background-image: url('../image/pattern-marquise.jpg');
```

Comment:
Where are these images? did you forget to commit them?

Reply:
No, I will add them soon with other useful assets. But anyway, I can't use them now, because I haven't yet figured out how to get username and faction name of current user from store (if it's ever possible now)

Reply:
You can get username from the socket (socket.name) and then you can find the faction of the user in `$state.factions.find(faction => faction.player === socket.name)`